policy/policy.html

```diff
@@ -0,0 +1,473 @@
+                        height="auto"
+                      /></div
+                  ></a>
+                </li>
+                <li class="dropdown__item">
+                  <a href="" class="dropdown__item-link"
+                    >ドクター
+                    <div class="dropdown__item-arrow">
+                      <img
+                        src="../assets/images/common/arrow/right-arrow.png"
+                        alt=""
+                        height="auto"
+                      /></div
+                  ></a>
+                </li>
+              </ul>
+            </li>
+            <li class="header__item pc__none">
+              <a href="" class="header__item-link"
+                ><span class="header__item-en">Info</span
+                ><span class="header__item-ja">会社情報</span>
+              </a>
+            </li>
+            <li class="header__item pc__none">
+              <a href="" class="header__item-link"
+                ><span class="header__item-en">Policy</span
+                ><span class="header__item-ja">ポリシー</span>
+              </a>
+            </li>
+          </ul>
+          <div class="header__btn-group">
+            <a href="" class="header__btn-sns">
+              <span class="header__btn-sns-en font__roboto">SNS</span>
+              <span class="header__btn-sns-ja">デポルターレジム instagram</span>
+            </a>
+            <a href="" class="header__btn-reserve">
+              <span class="header__btn-sns-en font__roboto">Reserve</span>
+              <span class="header__btn-sns-ja"
+                >体験・入会のご予約はこちらから</span
+              >
+            </a>
+          </div>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <section class="mv policy__mv">
+        <div class="mv__container">
+          <h1 class="mv__title">
+            <span class="mv__title-en font__anton">Info/Policy</span>
+            <span class="mv__title-ja">会社情報・ポリシー</span>
+          </h1>
+          <div class="mv__pic">
+            <img
+              src="../assets/images/policy/policy-mv.jpg"
+              alt="メインビジュアル"
+            />
+            <div class="mv__pic-bg"></div>
+          </div>
+          <div class="mv__button anim-box poyopoyo">
+            <a href="" class="button__link"
+              ><img
+                src="../assets/images/common/button/trial-button.png"
+                alt="無料体験ボタン"
+            /></a>
+          </div>
+        </div>
+      </section>
+      <section class="info">
+        <div class="info__container max-width">
+          <h2 class="section__heading">
+            <span class="section__heading-en font__anton">Info</span>
+            <span class="section__heading-ja">会社情報</span>
+          </h2>
+          <ul class="info__list">
+            <li class="info__item">
+              <h3 class="info__item-title">会社名</h3>
+              <h4 class="info__item-title-sub info__item-title-sub01">
+                有限会社デポルターレ（DEPORTARE）
```

Comment:
```suggestion
                有限会社 政和
```

ref. https://deportaregym.net/